tests/test_reusable_executor.py

```diff
@@ -949,6 +949,87 @@ def test_reusable_executor_reuse_true(self):
         executor4 = get_reusable_executor()
         assert executor4 is executor3
 
+    def test_faulthandler_enabled(self):
+        cmd = """if 1:
+            from loky import get_reusable_executor
+            from loky.process_executor import TerminatedWorkerError
+            import faulthandler
+            import sys
+
```

Comment:
Maybe a good idea to check the different cases with the multiprocessing debug?
```suggestion
import logging
from multiprocessing.util import log_to_stderr

log_to_stderr(logging.DEBUG)
```